Builds/Test.py

```diff
@@ -61,6 +66,41 @@ def powerset(iterable):
 IS_WINDOWS = platform.system().lower() == 'windows'
 IS_OS_X = platform.system().lower() == 'darwin'
 
+# CMake
+
+CMAKE_UNITY_TARGETS = ['rippled', 'rippled_classic']
+CMAKE_NONUNITY_TARGETS = ['rippled', 'rippled_unity']
+if IS_WINDOWS:
+    CMAKE_UNITY_CONFIGS = ['Debug', 'Release']
+    CMAKE_NONUNITY_CONFIGS = ['DebugClassic', 'ReleaseClassic']
+else:
+    CMAKE_UNITY_CONFIGS = []
+    CMAKE_NONUNITY_CONFIGS = []
+CMAKE_UNITY_COMBOS = { '' : [CMAKE_UNITY_TARGETS, CMAKE_UNITY_CONFIGS],
+    '.nounity' : [CMAKE_NONUNITY_TARGETS, CMAKE_NONUNITY_CONFIGS] }
```

Comment:
The `*_CONFIGS` are conditional, but I'm fine with doing that for the `*_TARGETS`.